packages/common/src/Enums.js

```diff
@@ -23,10 +23,10 @@ const LiquidationStatesEnum = {
 // Note that these are status translations AFTER a withdrawLiquidation method is called
 const PostWithdrawLiquidationRewardsStatusTranslations = {
   "0": "Liquidation deleted; All rewards have been withdrawn",
-  "3": "Dispute succeeded; Not all rewards have been withdrawn"
```

Comment:
this const Enum needs to support the new version of contracts wherein a withdrawal can be processed by any user, and state 4 is posible.

Reply:
After anyone calls `withdrawLiquidation()`, won't the state go to 0? So I think actually only state 0 is possible post-withdrawal?